lib/abci/handlers/query/verifyChainlockQueryHandlerFactory.js

```diff
@@ -0,0 +1,41 @@
+const { ChainLock } = require('@dashevo/dashcore-lib');
+const SimplifiedMNListStore = require('@dashevo/dashcore-lib/lib/deterministicmnlist/SimplifiedMNListStore');
+
+const {
+  abci: {
+    ResponseQuery,
+  },
+} = require('abci/types');
+
+/**
+ *
+ * @param {SimplifiedMNListStore} smlStore
+ * @return {verifyChainlockQueryHandler}
+ */
+function verifyChainlockQueryHandlerFactory(smlStore) {
```

Comment:
The dependency that you need calling `simplifiedMasternodeList` https://github.com/dashevo/js-drive/pull/398/files#diff-a89123a5ddbcd015a1045dd58b120fa3120cf1ff4d7c44583ee174395471f41fR192

Reply:
done